buildlib/pr/wire_compat.yml

```diff
@@ -90,22 +90,28 @@ jobs:
           UCX_LEGACY_PATH: $(System.DefaultWorkingDirectory)/$(ucx_legacy)
         displayName: compare tls performance characteristics
         condition: ne(variables.ucx_tag, 'v1.14.0')
-      - bash: buildlib/tools/test_wire_compat.sh $(test_dir)/client_server -a "-c tag"
+      - bash: |
+          export WIRE_COMPAT_TEST_PORT=$((10000 + 1000 * ${AZP_AGENT_ID}))
```

Comment:
maybe 
export WIRE_COMPAT_STAGE_ID=1

and in the script
`port=$((10000 + (1000 * ${AZP_AGENT_ID}) + (100* ${WIRE_COMPAT_STAGE_ID})))`

Reply:
fixed